### DISCUSSION TITLE
"linguist-vendored=false" not working

### DISCUSSION THREAD

**Discussion by danielb987** [community/community#21864] [New to GitHub]
<p>I’m trying to configure linguist to exclude files from my project. I have read this page, but maybe I don’t understand it:</p>
<p><a href="https://stackoverflow.com/questions/40659265/using-gitattributes-for-linguist-examples" rel="nofollow noopener">https://stackoverflow.com/questions/40659265/using-gitattributes-for-linguist-examples</a></p>
<p>My .gitattributes file is here:</p>
<p><a href="https://github.com/danielb987/JavaDiptraceAsciiLib/blob/master/.gitattributes" rel="nofollow noopener">https://github.com/danielb987/JavaDiptraceAsciiLib/blob/master/.gitattributes</a></p>
<p>The files src/examples/javadiptraceasciilib/*.asc are data files that are very big and of no interest for statistics.</p>
<p>The folders ‘findbugs’, ‘checkstyle’ and ‘coverage’ are third party software that’s included to validate the code and therefore should not be included in the statistics.</p>
<p>When I go to this page and looks at the languages, I see a lot of files that shouldn’t be there.</p>
<p><a href="https://github.com/danielb987/JavaDiptraceAsciiLib/search?l=Shell" rel="nofollow noopener">https://github.com/danielb987/JavaDiptraceAsciiLib/search?l=Shell</a></p>
<p>What am I doing wrong?</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/danielb987">@danielb987</a>,</p>
<p>Thanks for being part of the GitHub Community Forum!</p>
<p>Lines 6-8 in your .gitattributes file are actually doing the opposite of what you want. <code>linguist-vendored=false</code> means it will not be vendored; you want to have <code>linguist-vendored</code>.</p>
<p>Additionally, the only *.asc file being counted is <code>src/examples/javadiptraceasciilib/Layers.asc</code>, and that’s becuase it’s not actually included in the .gitattributes file.</p>
<p>I hope this helps!</p>

**Comment by danielb987** [comment]
<p>Thanks! Now it works as I want it to work.</p>
